tests/js/test-caseholder.js

```diff
@@ -0,0 +1,495 @@
+        wildcardDeepRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/wildcard/many/levels/deep"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        cookieSetRequest: {
+            type: "kettle.test.request.httpCookie",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/cookie"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        cookieReadRequest: {
+            type: "kettle.test.request.httpCookie",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        sessionRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        bodyParserRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        shutdownRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "{testEnvironment}.options.baseUrl",
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        }
+    },
+    modules: [
+        {
+            tests: [
+                {
+                    name: "Testing the 'static' router module (index content)...",
+                    type: "test",
+                    sequence: [
```

Comment:
As we discussed, this "sequence" can broken out into its own top-level piece of named configuration, and then the two repeated first sequence points supplied by a processing function rather than duplicating them. This will also reduce nesting in the file.